plugin.go

```diff
@@ -117,7 +118,10 @@ func (p *Plugin) teardown(ctx context.Context) error {
 }
 
 func (p *Plugin) getPreviewUrl() string {
-	return fmt.Sprintf("%s-%s-pr-%d.surge.sh", p.RepoOwner, p.RepoName, p.PullRequestId)
+	pattern := regexp.MustCompile(`(?i)[^a-z0-9\-]`)
```

Comment:
```suggestion
	pattern := regexp.MustCompile(`[^a-zA-Z0-9\-]`)
```

Reply:
Why do you prefer explicitly adding `A-Z` instead of using a [case-insensitive flag](https://pkg.go.dev/regexp/syntax)?

Reply:
It looked more explicit to me (at least a bit 😅) and it was one character short 😂. But we can keep it if you prefer.